src/NUnitFramework/framework/Internal/Execution/WorkItemBuilder.cs

```diff
@@ -41,11 +51,11 @@ public static class WorkItemBuilder
         /// <param name="filter">The filter to be used in selecting any child Tests.</param>
         /// <param name="recursive">True if child work items should be created and added.</param>
         /// <returns></returns>
-        static public WorkItem CreateWorkItem(ITest test, ITestFilter filter, bool recursive = false)
```

Comment:
This is another breaking change. For consistency we should do what we did for TimeoutCommand.